### PR TITLE
Moved API version to 2015-05-01-preview per feedback.

### PR DESCRIPTION
**Pull request by SuperScottz**

### CODE REVIEW

File: vm-linuxagent-upgrade/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+                    "subnet": {
+                        "id": "[variables('subnet1Ref')]"
+                    }
+                }
+            }
+            ]
+        }
+    },
+    {
+        "apiVersion": "2014-12-01-preview",
```
Comment (singhkay) [inline vm-linuxagent-upgrade/azuredeploy.json]: This is still using the old API version.

File: vm-linuxagent-upgrade/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+            "hardwareProfile": {
+                "vmSize": "[parameters('vmSize')]"
+            },
+            "osProfile": {
+                "computername": "[parameters('vmName')]",
+                "adminUsername": "[parameters('adminUsername')]",
+                "adminPassword": "[parameters('adminPassword')]"
+            },
+            "storageProfile": {
+                "sourceImage": {
```
Comment (singhkay) [inline vm-linuxagent-upgrade/azuredeploy.json]: Need to move to the new image specifying model. Please see this https://github.com/Azure/azure-quickstart-templates/blob/master/breaking-change.md

**Comment by singhkay** [comment]
@SuperScottz I still see the old API version specified in the template. Can you take a look and update?

**Comment by SuperScottz** [comment]
Sure, will do that, sorry for late.

**Comment by singhkay** [comment]
@SuperScottz Any update on this?

**Comment by singhkay** [comment]
Closing due to lack of updates. Feel free to reopen when ready.

**Pull request closed without merging**